website/docs/guides/core-to-cloud-1.md

```diff
@@ -0,0 +1,244 @@
+   - [AlloyDB](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) 
+   - [Amazon Redshift](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) 
+   - [Apache Spark](/docs/cloud/connect-data-platform/connect-apache-spark) 
+   - [Databricks](/docs/cloud/connect-data-platform/connect-databricks) 
+   - [Google BigQuery](/docs/cloud/connect-data-platform/connect-bigquery)
+   - [Microsoft Fabric](/docs/cloud/connect-data-platform/connect-microsoft-fabric)
+   - [PostgreSQL](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb)
+   - [Snowflake](/docs/cloud/connect-data-platform/connect-snowflake)
+   - [Starburst or Trino](/docs/cloud/connect-data-platform/connect-starburst-trino)
+
+2. You can verify your data platform connections by clicking the **Test connection** button in your deployment and development credentials settings.
+
+### Advanced configuration
+3. Use [OAuth connections](/docs/cloud/manage-access/set-up-snowflake-oauth), which enables secure authentication using your data platform’s SSO.  <Lifecycle status='enterprise' />
+
+## Git setup
+
+Your existing dbt project source code should live in a Git repository. In this step, you should connect your existing dbt project source code from Git to dbt Cloud.
+
+1. Ensure your dbt project is in a Git repository
+
+2. Once you’ve set up a dbt Cloud account, you can [connect and configure Git](/docs/cloud/git/git-configuration-in-dbt-cloud) in dbt Cloud to connect your Git repository:
+   - Connect with one of the [native integrations](/docs/cloud/git/git-configuration-in-dbt-cloud) in dbt Cloud (such as GitHub, GitLab, and Azure DevOps).
+   - [Import a git repository](/docs/cloud/git/import-a-project-by-git-url) from any valid git URL that points to a dbt project.
+
+### Advanced configuration
+3. Log into dbt Cloud using [OAuth connections](/docs/cloud/git/connect-github) to integrate with your source code platform. It automatically links to the repository using one of the native integrations set at the account level. <Lifecycle status='enterprise' />
+  
+  Set up groups for dbt project access with those configured for repository access to streamline permissions.
+
+## Developer setup
+
+This section highlights the development configurations you’ll need for your dbt Cloud project. The following categories are covered in this section:
+
+- [dbt Cloud environments](/guides/core-to-cloud-1?step=7#dbt-cloud-environments)
+- [Initial setup steps](/guides/core-to-cloud-1?step=7#initial-set-up-steps)
+- [Advanced configuration](/guides/core-to-cloud-1?step=7#initial-set-up-steps)
+- [dbt Cloud commands](/guides/core-to-cloud-1?step=7#dbt-cloud-commands)
+
+### dbt Cloud environments
+
+The concept of a `target` in dbt Core is the same as a [dbt Cloud environment](/docs/environments-in-dbt).
```

Comment:
Suggest reframin to switch the subject from Core (which they're leaving) to Cloud 
```suggestion
The concept of an [environment](/docs/environments-in-dbt) in dbt Cloud is the same as a `target` in dbt Core.
```